lifecycle-application/src/test/java/io/quarkus/ts/openshift/lifecycle/LifecycleApplicationOpenShiftIT.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.ts.openshift.lifecycle;
+
+import io.fabric8.kubernetes.api.model.PodList;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.common.OpenShiftTest;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@OpenShiftTest
+public class LifecycleApplicationOpenShiftIT {
+
+    private static final String DEFAULT_ARGUMENT = "-Djava.util.logging.manager=org.jboss.logmanager.LogManager";
+    private static final String RUNTIME_LABEL = "app.openshift.io/runtime";
+    private static final String RUNTIME_QUARKUS = "quarkus";
+
+    @TestResource
+    private OpenShiftClient openshiftClient;
+
+    @Test
+    public void shouldArgumentsContainDefaultArgument() {
+        when().get("/args")
+        .then().statusCode(200)
+                .body(containsString(DEFAULT_ARGUMENT));
+    }
+
+    @Test
+    public void shouldPrintMessagesFromQuarkusMain() {
+        PodList pods = openshiftClient.pods().withLabel(RUNTIME_LABEL, RUNTIME_QUARKUS).list();
+        assertEquals(1, pods.getItems().size(), "Found more than one pod with Quarkus runtime");
+
+        String actualLog = openshiftClient.pods().withName(pods.getItems().get(0).getMetadata().getName()).getLog();
+
+        assertTrue(actualLog.contains(Main.MESSAGE), "Pod log does not contain the expected message from Main.java");
```

Comment:
there should be also check for `Received arguments: ...` to ensure nothing gets sneaked in.

And if the tests fails it would be good to see the content of `actualLog`

Reply:
Changed.